source/_integrations/nuki.markdown

```diff
@@ -25,8 +25,36 @@ To add a Nuki bridge to your installation, you need to enable developer mode on
 
 {% include integrations/config_flow.md %}
 
+## Binary Sensors
+
+The integration provides a binary sensor if the Nuki door sensor is available. 
+
 ## Services
 
+### Service `lock.unlock`
```

Comment:
These services do not belong to the Nuki integration but are part of the Lock component of Home Assistant. It should not be documented here, it is documented with the Lock integration documentation.